pages/index.js

```diff
@@ -17,6 +17,9 @@ const Index = () => {
         />
       </Head>
       <HomePage />
+      <footer>
+        <p className="info-repo">The contents of this website are deployed from this <a href="https://github.com/Real-Dev-Squad/website-members" target="_blank" rel="noopener noreferrer">open sourced repo</a></p>
```

Comment:
@sumitd94 we are using SCSS-Modules so this won't work because all the classes/ids are scoped to their respective files. When you write `.info-repo`, it is converted to `.[filename]__[info-repo]___[randomHash]`. Look at the HomePage component to know how to properly import classNames from a SCSS file. 

Also, try creating a separate Footer component inside the `components` folder so that the Footer can be reused. 

This is how it is looking in my dev machine right now. Try sticking the footer at the bottom of the page, although it's not necessary. 
![Screenshot 2020-07-25 at 8 30 49 PM](https://user-images.githubusercontent.com/32963816/88459822-bc31bc80-ceb5-11ea-857c-c4917d4010ee.png)